tests/test_python_prefixes.py

```diff
@@ -103,8 +88,23 @@ def test_to_latin_numeral(value, expected):
     assert to_latin_numeral(value) == expected
 
 
-@pytest.mark.parametrize("value, keywords, expected_value, expected_system", get_value_and_bool_sys_test_data)
-def test_get_value_and_bool_sys(value, keywords, expected_value, expected_system):
-    bool_value, bool_system = get_value_and_bool_sys(value, keywords)
-    assert bool_value == expected_value
-    assert bool_system == expected_system
+value_str_test_data = [
+    (Value('hello Hedy'), 'hello Hedy'),
+    (Value(1, num_sys='Latin'), '1'),
+    (Value(True, bools={True: 'yes', False: 'no'}), 'yes'),
```

Comment:
Ah I think this is the problem. PyLint is saying that there's no `bools` keyword argument in the constructor. I think it's supossed to be `bool_sys`?